### PR TITLE
Update the text below the ace component with the current text

### PR DESCRIPTION
**Pull request by archaeron**
To show how one can listen to updates in the ace component

### CODE REVIEW

File: example/src/Main.purs [hunk 1]
```diff
@@ -72,7 +72,7 @@ ui = parentComponent render eval
                         )
                         Nothing
                 , initialState :
-                    { key : Nothing, editor : Nothing }
+                    initialAceState
```
Comment (garyb) [inline example/src/Main.purs]: Using `aceConstructor` for the whole slot definition is another option here. Not sure if it improves things though, what do you think?

Comment (archaeron) [inline example/src/Main.purs]: I'd like to try that, but I can't figure out how to transform the `SlotConstructor` that `aceConstructor` returns to an `HTML` element.

Comment (garyb) [inline example/src/Main.purs]: Ah right, you just wrap a `SlotConstructor` in a `Slot`, which is one of the `HTML` data constructors. The `slot` version is a smart constructor that makes the `SlotConstructor`-in-`Slot` for you.

Comment (archaeron) [inline example/src/Main.purs]: Thanks a lot, it worked :) And it looks better I'd say.

**Comment by garyb** [comment]
Thanks! :+1:

**Pull request merged**